foundation/auth/src/token_source/mod.rs

```diff
@@ -19,7 +20,7 @@ pub mod service_account_token_source;
 pub mod external_account_source;
 
 #[async_trait]
-pub trait TokenSource: Send + Sync + Debug {
+pub trait GcloudTokenSource: Send + Sync + Debug {
```

Comment:
Could you please use `GoogleCloudTokenSource` instead of `GcloudTokenSource`?

Reply:
done